api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/oauth/OAuthProperties.java

```diff
@@ -73,6 +73,11 @@
      */
     private UserProfileViewTypes userProfileViewType = UserProfileViewTypes.NESTED;
 
+    /**
+     * Configurable response_type parameter in pac4j.
```

Comment:
Describe what the parameter does.

Reply:
I'm changing it with a commit. Meanwhile, [here is a link](https://developer.okta.com/blog/2018/05/24/what-is-the-oauth2-implicit-grant-type) for reference.